src/Dfc.CourseDirectory.Core/Models/Region.cs

```diff
@@ -4,842 +4,6 @@ namespace Dfc.CourseDirectory.Core.Models
-                }
-            },
-            new Region
-            {
-                Id = "E12000009",
-                Name = "South West",
-                SubRegions = new List<Region>
-                {
-                    new Region
-                    {
-                        Id = "E06000022",
-                        Name = "Bath and North East Somerset",
-                    },
-                    new Region
-                    {
-                        Id = "E06000023",
-                        Name = "Bristol",
-                    },
-                    new Region
-                    {
-                        Id = "E06000024",
-                        Name = "North Somerset",
-                    },
-                    new Region
-                    {
-                        Id = "E06000025",
-                        Name = "South Gloucestershire",
-                    },
-                    new Region
-                    {
-                        Id = "E06000026",
-                        Name = "Plymouth",
-                    },
-                    new Region
-                    {
-                        Id = "E06000027",
-                        Name = "Torbay",
-                    },
-                    new Region
-                    {
-                        Id = "E06000028",
-                        Name = "Bournemouth",
-                    },
-                    new Region
-                    {
-                        Id = "E06000029",
-                        Name = "Poole",
-                    },
-                    new Region
-                    {
-                        Id = "E06000030",
-                        Name = "Swindon",
-                    },
-                    new Region
-                    {
-                        Id = "E06000052",
-                        Name = "Cornwall",
-                    },
-                    new Region
-                    {
-                        Id = "E06000053",
-                        Name = "Isles of Scilly",
-                    },
-                    new Region
-                    {
-                        Id = "E06000054",
-                        Name = "Wiltshire",
-                    },
-                    new Region
-                    {
-                        Id = "E10000008",
-                        Name = "Devon",
-                    },
-                    new Region
-                    {
-                        Id = "E10000009",
-                        Name = "Dorset",
-                    },
-                    new Region
-                    {
-                        Id = "E10000013",
-                        Name = "Gloucestershire",
-                    },
-                    new Region
-                    {
-                        Id = "E10000027",
-                        Name = "Somerset",
-                    }
-                }
-            }
-        };
-
         public string Id { get; set; }
```

Comment:
Suggest reworking as a "record" type to prevent mutation whilst in cache. (Old style with ctor and `{ get; }` only props?